apply_diff.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# usage `./apply_diff.sh 500`
+MODULE_COUNT=$1
+declare -A MAX_MEMORY=(
+  ["50"]="400m"
+  ["100"]="600m"
+  ["200"]="1300m"
+  ["500"]="4000m"
+  ["1000"]="9000m"
```

Comment:
We've measured these as working with the latest 8.0 nightly:

200 - 1100m
500 - 3200m
1000 - 6800m

Reply:
Changed it.